transform/transformers/mbs_transformer.py

```diff
@@ -180,10 +189,14 @@ def transform(self):
             "40": self.round_mbs(self.response["data"].get("40")),
             "49": self.round_mbs(self.response["data"].get("49")),
             "90": self.round_mbs(self.response["data"].get("90")),
-            "50": self.response["data"].get("50"),
+            "50": MBSTransformer.convert_str(self.response["data"].get("50")),
```

Comment:
I don't think this method name makes it clear what we're converting the string into.